bioscrape/simulator.pyx

```diff
@@ -621,6 +621,293 @@ cdef class SSAResult:
             return self.py_get_result()
 
 
+    def py_empirical_distribution(self, start_time = None, species = None, Model = None, final_time = None, max_counts = None):
+        
```

Comment:
Can you add docstrings here for a general description of what this function can do. I think you already have comments explaining things which could be moved here.

Reply:
and if possible, same for other functions here too.

Reply:
Thanks! Did the commit not go through though? I can't see the new commit..